web/filament-js/filament.d.ts

```diff
@@ -34,6 +34,7 @@ export const assets: {[url: string]: Uint8Array};
 export type float2 = glm.vec2|number[];
 export type float3 = glm.vec3|number[];
 export type float4 = glm.vec4|number[];
+export type double4 = glm.vec4|number[];
```

Comment:
Technically we don't need this particular TypeScript definition, but I guess it is a potentially useful hint to humans that means: "double precision will be preserved all the way down"